release/README.md

```diff
@@ -2,7 +2,11 @@
 
 This repository contains the docker image for odfe-pipes.
 
-## Running a build
+## Running a docker build
 
-To build an image check out the corresponding branch for the version and run `./gradlew :release:docker`
+To build an image, check out the corresponding branch for the version and run `./gradlew clean :release:docker:docker -Prelease`
+
+## Running a tar build
+
+To build a tar, check out the corresponding branch for the version and run `./gradlew clean :release:linux:linuxDistTar -Prelease`
```

Comment:
1. Since .gradlew is in `../`, both the release task fail to run, can we change the instructions or the command.
2. Add instructions on "Where do I find the docker-image or tar?"